src/client/java/README.md

```diff
@@ -70,7 +70,7 @@ When run with Hadoop yarn, you need to add below configuration to core-site.xml.
 ```xml
 <property>
 <name>fs.AbstractFileSystem.daos.impl</name>
-<value>com.intel.daos.hadoop.fs.DaosAbsFsImpl</value>
+<value>io.daos.fs.hadoop.DaosAbsFsImpl</value>
```

Comment:
We also need to update daos-java-api to daos-client at the beginning of this file. 
```
There are two submodules, daos-java-api and hadoop-daos.
### daos-java-api
```

Reply:
good catch.